### DISCUSSION TITLE
Where to put my system design tutorial?

### DISCUSSION THREAD

**Discussion by lahin31** [me-shaon/bangla-programming-resources#109] [Q&A] [answered]
I've created a System Design tutorial in Bangla https://github.com/lahin31/system-design-bangla. I would like to know where to put it.

**Comment by me-shaon** [comment, marked as answer]
Please create a section called 'System design' here: https://github.com/me-shaon/bangla-programming-resources#%E0%A6%85%E0%A6%A8%E0%A7%8D%E0%A6%AF%E0%A6%BE%E0%A6%A8%E0%A7%8D%E0%A6%AF-1
